### PR TITLE
Show last deploy operation user and time, current git ref on app show

### LINKED ISSUE #73: Properly handle app.lastDeployOperation and app.currentImage
These are of type operation and image, respectively, and are embedded by HAL.
The HAL adapter needs to be updated to handle embedded properties that have a different name (eg 'last_deploy_operation') than their type (eg 'operation').

See https://github.com/201-created/ember-data-hal-9000/issues/7

### PR DESCRIPTION
**Pull request by bantic**
* add `format-date` helper
  * add `iso-8601-timestamp` transform
  * add `image` model
  * DRY common model dependencies for unit tests
  * test that an app's `currentImage` and `lastDeployOperation` can be sideloaded

@mixonic for you
fixes #73 #86

### CODE REVIEW

**Comment by bantic** [comment]
looks like 

image

**Comment by bantic** [comment]
also format utc timestamps on app/db activity pages. fixes #87.

looks like

image

File: app/helpers/format-date.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { monthNames } from 'diesel/utils/dates';
+
+export function formatDate(date) {
+  if (!date) { return; }
+
+  Ember.assert('format-date must be called with an instanceof Date',
+               date instanceof Date);
+
+  let monthName = monthNames[ date.getMonth() ];
+  return `${monthName} ${date.getDate()}, ${date.getFullYear()}`;
```
Comment (mixonic) [inline app/helpers/format-date.js]: @sandersonet @fancyremarker check out this use of ES2015 string interpolation. These are also multi-line capable.

File: tests/unit/models/operation-test.js [hunk 2]
```diff
@@ -4,22 +4,13 @@ import {
 } from 'ember-qunit';
 import { stubRequest } from "../../helpers/fake-server";
 import Ember from 'ember';
+import modelDeps from '../../support/common-model-dependencies';
```
Comment (mixonic) [inline tests/unit/models/operation-test.js]: Might want to refactor these to be called `stackChildModelDeps` or something later. `common` implies common to all models, and these are really constrained to children of stack.

**Pull request merged**